alf/algorithms/ddpg_algorithm_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2020 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl import logging
+import functools
+import torch
+import torch.distributions as td
+import unittest
+
+import alf
+from alf.algorithms.config import TrainerConfig
+from alf.algorithms.rl_algorithm import RLAlgorithm
+from alf.algorithms.ddpg_algorithm import DdpgAlgorithm
+from alf.algorithms.rl_algorithm_test import MyEnv
+from alf.data_structures import StepType, TimeStep
+from alf.environments.suite_unittest import PolicyUnittestEnv, ActionType
+from alf.networks import ActorNetwork, CriticNetwork
+from alf.algorithms.ppo_algorithm_test import unroll
+from alf.utils import common, dist_utils, tensor_utils
+from alf.utils.math_ops import clipped_exp
+
+
+class DDPGAlgorithmTest(alf.test.TestCase):
+    def test_ddpg_algorithm(self):
+        num_env = 1
+        config = TrainerConfig(
+            root_dir="dummy",
+            unroll_length=1,
+            mini_batch_length=2,
+            mini_batch_size=64,
+            initial_collect_steps=500,
+            whole_replay_buffer_training=False,
+            clear_replay_buffer=False,
+            num_envs=num_env,
+        )
+        env_class = PolicyUnittestEnv
+        steps_per_episode = 13
+        env = env_class(
+            num_env, steps_per_episode, action_type=ActionType.Continuous)
+
+        eval_env = env_class(
+            num_env, steps_per_episode, action_type=ActionType.Continuous)
```

Comment:
use a large batch for eval_env can make the test more robust.

Reply:
The eval_env batch is increased in the test case.